vim_onenote.ahk

```diff
@@ -117,77 +216,77 @@ Send, {End}{Enter}
 Gosub InsertMode
 return
 
+r::
+    send +{right}
+    gosub, InsertMode
+    ; Wait for single key to be pressed, sends that key and returns to normal
+    input, inp, V E L1
+    send {left}
+    gosub, NormalMode
+return
+
 ; undo
 u::Send, ^z
-return 
 ; redo.
 ^r::Send, ^y
-return 
 
-+G:: Send, ^{End}
 ; G goto to end of document
-return
+sG(){
+    Send, ^{End}
+    }
++G::sG()
 
-g::
+g(){
 ; TBD - Design a more generic way to implement the <command> <motion> pattern. For now hardcode dw. 
 if IsLastKey("g")
-{
     ;gg - Go to start of document
     Send, ^{Home}
-    return
 }
-return
+g::g()
```

Comment:
I don't understand this at all, sorry. If it's still relevant in light of other replies, could you try re-wording it?